File: 2022/03/1.py
import re

data = open("i.txt").read().split("\n")

n = 0
for d in data:
    s1 = d[:len(d)//2]
    s2 = d[len(d)//2:]
    i = list(set(s1).intersection(s2))[0]

    if ord(i) < 97:
        n+= ord(i)-65+27
    else:
        n+= ord(i)-96

print(n)

File: 2022/03/js/1.js
const fs = require("fs");
let data = fs.readFileSync("i.txt").toString().split(/\r?\n/);
let n = 0

console.log(n);

console.log(data.reduce((a1, e) => a1 + e.substring(0, e.length / 2).split("").reduce((a2, c) => {
    if (a2 != 0 || e.substring(e.length / 2).indexOf(c) == -1) return a2;
    if (c.charCodeAt(0) < 97) return c.charCodeAt(0) - 65 + 27;
    return c.charCodeAt(0) - 96;
}, 0), 0));